src/utils/connect/fetch.ts

```diff
@@ -0,0 +1,11 @@
+import { LibraryError } from '../errors';
+
+export default (typeof fetch !== 'undefined' && fetch) ||
+  (typeof globalThis !== 'undefined' && globalThis.fetch) ||
```

Comment:
```suggestion
  (typeof globalThis !== 'undefined' && globalThis.fetch.bind(globalThis) ||
```

Similar to `window` case on the next line. E.g., browser service workers support `fetch` and `globalThis`, but don't have `window`.